controls/scheduler/localization.md

```diff
@@ -22,6 +22,67 @@ The table below shows the localization keys:
 | `Scheduler_MonthViewDefinition_Title` | `Month` |
 | `Scheduler_Today_Header_Text` | `Today` |
 | `Scheduler_Empty_ViewDefinition_Text` | `Please declare a View definition` |
+| `Scheduler_EditAppointmentDialog_Subject` | `Subject`|
+| `Scheduler_EditAppointmentDialog_StartTime` | `Start Time` |
+| `Scheduler_EditAppointmentDialog_EndTime` | `End Time` |
+| `Scheduler_EditAppointmentDialog_AllDay` | `All Day` |
+| `Scheduler_EditAppointmentDialog_Repeat` | `Repeat` |
+| `Scheduler_EditAppointmentDialog_Description` | `Description` |
+| `Scheduler_EditAppointmentDialog_Save` | `Save` |
+| `Scheduler_EditAppointmentDialog_Cancel` | `Cancel` |
+| `Scheduler_EditAppointmentDialog_Edit` | `Edit` |
+| `Scheduler_EditAppointmentDialog_Edit` | `Does not repeat` |
+| `Scheduler_EditAppointmentDialog_Repeat_Daily` | `Repeat daily` |
+| `Scheduler_EditAppointmentDialog_Repeat_Weekly` | `Repeat weekly` |
+| `Scheduler_EditAppointmentDialog_Repeat_Monthly` | `Repeat monthly` |
+| `Scheduler_EditAppointmentDialog_Repeat_Yearly` | `Repeat yearly` |
+| `Scheduler_EditAppointmentDialog_Repeat_Edit` | `Edit` |
+| `Scheduler_EditRecurrenceDialog_Repeat` | `Repeat` |
+| `Scheduler_EditRecurrenceDialog_Every` | `Every` |
+| `Scheduler_EditRecurrenceDialog_On` | `On` |
+| `Scheduler_EditRecurrenceDialog_End` | `End` |
+| `Scheduler_EditRecurrenceDialog_Repeat_Daily` | `Daily` |
+| `Scheduler_EditRecurrenceDialog_Repeat_Weekly` | `Weekly` |
+| `Scheduler_EditRecurrenceDialog_Repeat_Monthly` | `Monthly` |
+| `Scheduler_EditRecurrenceDialog_Repeat_Yearly` | `Yearly` |
+| `Scheduler_EditRecurrenceDialog_End_On_This_Day` | `on this day` |
```

Comment:
Please, double-check the capitalization for all lowercase entries.

Reply:
These are correct